cfe_internal/enterprise/CFE_hub_specific.cf

```diff
@@ -570,6 +570,27 @@ bundle agent cfe_internal_enterprise_maintenance
         if => isgreaterthan( $(enterprise_maintenance_bundle_count), 0 );
 }
 
+bundle agent cfe_internal_exported_report_location
+# @brief Ensure that exported reports are in the correct directory
+{
+  meta:
+    am_policy_hub.enterprise_edition::
+      "tags" slist => { "enterprise_maintenance" };
+
+  files:
+
+    am_policy_hub.enterprise_edition::
+
+      "$(cfe_internal_hub_vars.public_docroot)/tmp/." -> { "ENT-7465" }
```

Comment:
in 3.18 (htdocs/public/tmp) and 3.15 (htdocs/tmp) tmp has different location, I'd say to add variable and use it as bundle:
 ```
 vars:
    "tmp_directories_list" slist => {"$(cfe_internal_hub_vars.public_docroot)/tmp/.", "$(cfe_internal_hub_vars.docroot)/tmp/."};
```